packages/devextreme/testing/tests/DevExpress.ui.widgets/chat.tests.js

```diff
@@ -371,6 +370,65 @@ QUnit.module('renderMessage', moduleConfig, () => {
 
         assert.strictEqual($(lastBubble).text(), text);
     });
+
+    QUnit.test('Message list should not run invalidate if 1 new message has been added to items', function(assert) {
+        const invalidateStub = sinon.stub(this.instance._messageList, '_invalidate');
+
+        const { items } = this.instance.option();
+        const newMessage = {
+            timestamp: NOW,
+            author: userFirst,
+            text: 'NEW MESSAGE',
+        };
+
+        this.instance.option({ items: [...items, newMessage] });
+
+        assert.strictEqual(invalidateStub.callCount, 0);
+    });
+
+    QUnit.test('Message list should run renderMessage if 1 new message has been added to items', function(assert) {
+        const renderMessageStub = sinon.stub(this.instance._messageList, '_renderMessage');
```

Comment:
This test is too synthetic for me. Calling of a private `_renderMessage` in some scenario does not give us any helpful information to prove scenario works correctly.

Let's add a usual test checking that necessary markup is rendered

Reply:
[ede0a5cf8649c6d51b031fea534215b397380fac]